directories/src/redux/epics/treasures-epics.js

```diff
@@ -0,0 +1,63 @@
+import { Observable } from "rxjs";
+import { combineEpics } from "redux-observable";
+import moment from "moment";
+import _ from "lodash";
+
+import nodeActions from "../actions/node-actions";
+
+import brokerNode from "../services/broker-node";
+import iota from "../services/iota";
+import ethereum from "../services/ethereum";
+
+const claimTreasureEpic = (action$, store) => {
+  return action$.ofType(nodeActions.NODE_CLAIM_TREASURE).mergeMap(action => {
+    return Observable.fromPromise(brokerNode.treasures())
+      .mergeMap(({ data }) => {
+        const {
+          receiverEthAdd,
+          treasure: { genesisHash, sectorIdx, ethAddr, ethKey }
```

Comment:
I think we potentially don't need this ethAddr and ethKey, unless my understanding of the architecture is flawed.  The brokernode will need the website owner's ETH address but won't need any other ethereum-related information afaik.  

We'll also need to include the file size or number of chunks.  The broker will need this information to know when to stop building the data map, if the sector happens to be the last sector.